app/src/forms/form/routes.js

```diff
@@ -15,7 +14,7 @@ routes.param('formId', validateParameter.validateFormId);
 routes.param('formVersionDraftId', validateParameter.validateFormVersionDraftId);
 routes.param('formVersionId', validateParameter.validateFormVersionId);
 
-routes.get('/', keycloak.protect(`${config.get('server.keycloak.clientId')}:admin`), async (req, res, next) => {
+routes.get('/', jwtService.protect('admin'), async (req, res, next) => {
```

Comment:
Previously the name of the administrator role was configurable. Any concern that another jurisdiction would want the administrator role to be named something other than `admin`?

Reply:
the previous syntax was for hierarchy to check through `resouce_access` which was key: [ roles ], with the key being a KC service client. in this PR, roles are just an array and we only have one service client. if the token isn't for our client it is rejected BEFORE we look for roles (now in `client_roles`). of course, if we change our token verification to not care about the client, we will need to adjust our `protect` middleware to not only look for the role but the client/audience too. TLDR - not a concern at the moment.